src/main/resources/static/bindings.json

```diff
@@ -51,6 +51,30 @@
                                         }
                                     ]
                                 }
+                            },
+                            {
+                                "app_id": "spring-boot-example",
+                                "location": "fileNotification",
+                                "label": "fileNotification",
+                                "form": {
+                                    "title": "Sends notidication when figma file was commented",
+                                    "icon": "icon.png",
+                                    "submit": {
+                                        "path": "/webhook/subscribe/fileComment",
```

Comment:
hmm i'm not sure that our command should be under webhook  path /webhook/subscribe/fileComment
as the result that  /webhook/ path used for webhooks that MM server send to us it could be a little tricky .
Please think about changing this path  to something like /subscribe/fileComment . 
And please check do we need also this in a path /fileComment . 
Is there any difference when bot user will subscribe to fileComment or another one ? can we have path just like: 
"path": "/subscribe"